tests/integration_tests/test_ert_jobs.py

```diff
@@ -39,3 +44,79 @@ def test_export_connection_status(testdata_folder: Path, tmp_path: Path) -> None
 
     df = pd.read_parquet(output_file)
     assert list(df.columns) == ["DATE", "WELL", "I", "J", "K", "OP/SH"]
+
+
+@pytest.fixture
+def expected_jobs():
+    """dictionary of installed jobs with location to config"""
+    config_location = (
+        Path(__file__).absolute().parent.parent.parent
+        / "webviz_subsurface"
+        / "ert_jobs"
+    )
+    expected_job_names = ["EXPORT_CONNECTION_STATUS"]
```

Comment:
This will be named WELL_CONNECTION_STATUS

Reply:
We will rename in #689.